### ISSUE TITLE
Support for Meteor < 1.2.x

### ISSUE THREAD

**Issue by micahalcorn** [peerlibrary/meteor-reactive-publish#3]
I'm using [matb33:collection-hooks](https://github.com/matb33/meteor-collection-hooks) in an app, and if I try to use autorun in a publish function I get the following:

``` bash
Exception from sub foos id .............. TypeError: Object [object Object] has no method 'autorun'
at [object Object].<anonymous> (app/reactive-composite-hooks.js:35:10)
...
```

This happens in Meteor 1.1.x but not Meteor 1.2.x.

**Comment by mitar** [comment]
No idea, @matb33, is there some special issue how we are all wrapping original publish? Does this package has to do anything special for it to be compatible?

I know that in my [other package](https://github.com/peerlibrary/meteor-related/blob/master/server.coffee#L33) I make sure I copy any new/extra stuff over, just in case.

**Comment by micahalcorn** [comment]
**Correction**

The package is not installing in my Meteor 1.1.x app due to the inability to recognize `isobuild:isopack-2`. The issue has nothing to do with `collection-hooks`, sorry.

**Comment by mitar** [comment]
I have no idea where this error is coming from. This package does not require it. Also in `package.js` I am not requiring Meteor 1.2.

Maybe you should report a bug to Meteor?

**Comment by mitar** [comment]
It seems this bug is related: https://github.com/meteor/meteor/issues/5347

**Comment by mitar** [comment]
Seems Meteor is breaking this with Meteor 1.2 in a backwards incompatible way. So packages which use CoffeeScript and are published with Meteor 1.2 cannot be used with old Meteor versions.

For now I propose you simply clone this package source as a git submodule into your `packages` directory in your app and it should work for you.

I will look into publishing all packages with older Meteor version.

**Comment by mitar** [comment]
I published a new version made with an older Meteor version. It should work now for you.
